### PR TITLE
[BACKLOG-6012] - skeleton for hide measure and hide attribute annotat…

### PR DESCRIPTION
**Pull request by kurtwalker**
…ions

### CODE REVIEW

**Comment by kurtwalker** [comment]
@rfellows if it helps to have this now, go ahead and pull.  if not we can wait for all the functionality

File: src/org/pentaho/agilebi/modeler/models/annotations/HideMeasure.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import org.pentaho.di.i18n.BaseMessages;
+import org.pentaho.metastore.api.IMetaStore;
+import org.pentaho.metastore.persist.MetaStoreAttribute;
+import org.w3c.dom.Document;
+
+public class HideMeasure extends AnnotationType {
+  protected static final Class<?> MSG_CLASS = BaseModelerWorkspaceHelper.class;
+
+  public static final String NAME_ID = "name";
+  public static final String NAME_NAME = "Level Name";
```
Comment (rfellows) [inline src/org/pentaho/agilebi/modeler/models/annotations/HideMeasure.java]: should this be "Measure Name"?

Comment (kurtwalker) [inline src/org/pentaho/agilebi/modeler/models/annotations/HideMeasure.java]: yes

**Comment by kurtwalker** [comment]
@rfellows fixed

**Comment by rfellows** [comment]
@kurtwalker this will help. I'll be able to create the objects in analyzer (server-side) now without compile issues.

**Comment by lucboudreau** [comment]
Build Commands:
==============
Build:
------
1. rm -r ~/.kettle; rm -r ~/.pentaho; echo removed hidden folders
2. ant -Dpentaho.coding.standards.version=1.0.1 -Djunit.jvmargs=-XX:PermSize=256m -f build.xml clean-all resolve jacoco jacoco-integration checkstyle publish-local

Cleanup:
-------
rm -r ~/.ivy2/local || echo no publish local to remove

Changed files:
==============
  src/org/pentaho/agilebi/modeler/messages/messages_en_US.properties
  src/org/pentaho/agilebi/modeler/models/annotations/HideAttribute.java
  src/org/pentaho/agilebi/modeler/models/annotations/HideMeasure.java
  src/org/pentaho/agilebi/modeler/models/annotations/ModelAnnotation.java
  test-src/org/pentaho/agilebi/modeler/models/annotations/HideAttributeTest.java
  test-src/org/pentaho/agilebi/modeler/models/annotations/HideMeasureTest.java
  test-src/org/pentaho/agilebi/modeler/models/annotations/ModelAnnotationTest.java










Unit test coverage change:
============
org.pentaho.agilebi.modeler.models.annotations.HideMeasure:
 * linePercentCoverage :64.29%
 * methodPercentCoverage :58.33%
 * complexityPercentCoverage :58.33%
 * instructionPercentCoverage :80.00%
    
org.pentaho.agilebi.modeler.models.annotations.HideAttribute:
 * linePercentCoverage :75.00%
 * methodPercentCoverage :68.75%
 * complexityPercentCoverage :68.75%
 * instructionPercentCoverage :86.96%
    
org.pentaho.agilebi.modeler.models.annotations.ModelAnnotation.Type:
 * instructionPercentCoverageChange : +2.06%
 * linePercentCoverageChange : +2.51%
    
Integration test coverage change:
============
org.pentaho.agilebi.modeler.models.annotations.ModelAnnotation.Type:
 * instructionPercentCoverageChange : +3.14%
 * linePercentCoverageChange : +3.01%

**Pull request merged**